demos/tests/run_tests.py

```diff
@@ -50,8 +53,26 @@ def parse_args():
         help='list of demos to run tests for (by default, every demo is tested)')
     parser.add_argument('--mo', type=Path, metavar='MO.PY',
         help='Model Optimizer entry point script')
+    parser.add_argument('--devices', default="CPU, GPU",
```

Comment:
I suggest using just `,` without a spcae as separator to follow other tools style, like https://github.com/opencv/open_model_zoo/blob/06c73b9662118efcd66f363760bf20898f0cb763/tools/downloader/downloader.py#L194

Reply:
Actually no. This will make it harder to parse device in case of `MULTI:CPU,GPU`. So I'm fine with space.

Reply:
I'm also find with spaces, but then I am not fine with commas. If you want to use spaces for separation, then just use spaces.